Makefile

```diff
@@ -84,7 +85,10 @@ pkg-remove:
 pkg-list:
 	@$(GOPATH)/bin/govendor list
 
-install: gomake-all
+# Builds minio and installs it to $GOPATH/bin.
+install: build
+	@echo "Installing minio at $(GOPATH)/bin/minio"
+	@cp $(PWD)/minio $(GOPATH)/bin/minio
```

Comment:
users might type `minio server export` in the source directory which is not preferred. Telling users to do `minio server ~/export` will also not be correct for windows users.

We could just leave it at this. So users will type "minio -h" for help